### DISCUSSION TITLE
Footnotes improvement.

### DISCUSSION THREAD

**Discussion by Pates93** [statisticssweden/PxWeb#181] [Ideas]
Hi all.

When we changed our PxWeb into 2021v1, I was approached by an co worker about one thing she thought was annoying with PxWeb. 

When you select a big amount of data, the table will become very large and the footnotes will be hard to find, because you have to scroll all the way to the bottom of the page to find the footnotes. She also thought it was strange that you could not find the footnotes under "About table". 

After thinking about this, I added some javascript the the site and added a new element called "Footnotes" under "About table".

![image](https://user-images.githubusercontent.com/37102638/138263089-c98e3e1f-6a44-40ea-92d1-5ed1fcd93bd8.png)

If you click the footnotes, the site will scroll down to the footnotes and will expand the footnotes. 

Maybe this is an idea to improve PxWeb? Let me know what you think.

If you want to try it out, you can go to our statbank, select a big amount of data and try the button
 
https://statbank.hagstova.fo/pxweb/en/H2/ 

I'm not sure though if this is in line with WCAG rules.

**Comment by likp** [comment]
This is a interesting solution on the problem. As you already seam to have implement this would you like to contribute this back to PxWeb as    an pull request?
